test/mongodb.test.js

```diff
@@ -345,7 +345,7 @@ describe('mongodb connector', function() {
       });
       ds.ping(function(err) {
         (!!err).should.be.True();
-        err.message.should.match(/failed to connect to server/);
+        err.code.should.eql('ECONNREFUSED');
```

Comment:
Please move `ECONNREFUSED ` assertion changes to a standalone commit, or even to a standalone pull request (to get them landed quickly).